tests/test_data.py

```diff
@@ -88,8 +88,8 @@ def test_read_rts(builder, filename, rt_format, expected):
     [PandasDataBuilder(), "Alkanes_20210325.csv", range(1100, 4100, 100)],
     [PandasDataBuilder(), "Alkanes_20210325.tsv", range(1100, 4100, 100)],
     [PandasDataBuilder(), "Alkanes_ri.csv", range(1100, 4100, 100)],
-    [PandasDataBuilder(), "Alkanes_retention_index.csv", range(1100, 4100, 100)],
-    [MatchMSDataBuilder(), "recetox_gc-ei_ms_20201028.msp", [2876, 2886.9, 1827.1, 1832.9, 1844.4, 1501, 1528.3, 2102.7, 2154.5, 2207.5]]
+    [PandasDataBuilder(), "Alkanes_retention_index.csv", range(1100, 4100, 100)]
+    #[MatchMSDataBuilder(), "recetox_gc-ei_ms_20201028.msp", [2876, 2886.9, 1827.1, 1832.9, 1844.4, 1501, 1528.3, 2102.7, 2154.5, 2207.5]]

```

Comment:
Are we playing to reintroduce this later?

Reply:
Eventually - I thought about adding an exception but at the moment, I don't see how we should handle data that has no RT